### PR TITLE
Azure Key Vault - Fix issue 1128- pfx file path guidance for the az kv secret store

### PR DESCRIPTION
**Pull request by msfussell**
Fix for Issue #1128

### CODE REVIEW

**Review by georgestevens99 — APPROVED** [review]
This looks good.

**Comment by msfussell** [comment]
@orizohar @AaronCrawfis - Could one of you also approved this PR?

File: daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
   - name: spnClientId
     value: "[your_service_principal_app_id]"
   - name: spnCertificateFile
-    value : "[pfx_certificate_file_local_path]"
+    value : "[pfx_certificate_file_fully_qualified_local_path"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md]: Needs closing ]

File: daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md [hunk 2]
```diff
@@ -151,22 +151,22 @@ The above example uses secrets as plain strings. It is recommended to use a loca
       - name: spnClientId
         value: "[your_service_principal_app_id]"
       - name: spnCertificateFile
-        value : "[pfx_certificate_file_local_path]"
+        value : "[pfx_certificate_file_fully_qualified_local_path"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md]: Needs closing ]

File: daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md (hunk 1 above)
Comment (msfussell) [reply daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md]: Fixed! Well spotted

File: daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md (hunk 2 above)
Comment (msfussell) [reply daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md]: Fixed! Well spotted

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**